src/proposals/20251002/SparkEthereum_20251002.t.sol

```diff
@@ -0,0 +1,504 @@
+
+        _assertRateLimit(transferKey, 200_000e18, 200_000e18 / uint256(1 days));
+
+        _testTransferAssetIntegration(
+            SYRUP,
+            Ethereum.ALM_OPS_MULTISIG,
+            Ethereum.ALM_CONTROLLER,
+            200_000e18,
+            200_000e18
+        );
+    }
+
+    function test_ETHEREUM_usdsTransfers() public onChain(ChainIdUtils.Ethereum()) {
+        uint256 foundationUsdsBalanceBefore = IERC20(Ethereum.USDS).balanceOf(Ethereum.SPARK_FOUNDATION);
+        uint256 groveUsdsBalanceBefore      = IERC20(Ethereum.USDS).balanceOf(GROVE_SUBDAO_PROXY);
+        uint256 sparkUsdsBalanceBefore      = IERC20(Ethereum.USDS).balanceOf(Ethereum.SPARK_PROXY);
+
+        assertEq(sparkUsdsBalanceBefore,      32_163_684.945801365846236778e18);
+        assertEq(foundationUsdsBalanceBefore, 292_388.004e18);
+        assertEq(groveUsdsBalanceBefore,      30_654e18);
+
+        executeAllPayloadsAndBridges();
+
+        assertEq(IERC20(Ethereum.USDS).balanceOf(Ethereum.SPARK_PROXY),      sparkUsdsBalanceBefore - AMOUNT_TO_GROVE - AMOUNT_TO_SPARK_FOUNDATION);
+        assertEq(IERC20(Ethereum.USDS).balanceOf(Ethereum.SPARK_FOUNDATION), foundationUsdsBalanceBefore + AMOUNT_TO_SPARK_FOUNDATION);
+        assertEq(IERC20(Ethereum.USDS).balanceOf(GROVE_SUBDAO_PROXY),        groveUsdsBalanceBefore + AMOUNT_TO_GROVE);
+    }
+
+    function test_ETHEREUM_symbioticConfiguration() public onChain(ChainIdUtils.Ethereum()) {
+        address NETWORK   = Ethereum.SPARK_PROXY;
+        address OWNER     = Ethereum.SPARK_PROXY;
+        address OPERATOR  = Ethereum.SPARK_PROXY;
+
+        INetworkRestakeDelegator delegator = INetworkRestakeDelegator(NETWORK_DELEGATOR);
+        INetworkRegistry networkRegistry   = INetworkRegistry(NETWORK_REGISTRY);
+        IOperatorRegistry operatorRegistry = IOperatorRegistry(OPERATOR_REGISTRY);
+
+        bytes32 subnetwork = bytes32(uint256(uint160(NETWORK)) << 96 | 0);  // Subnetwork.subnetwork(network, 0)
+
+        assertEq(networkRegistry.isEntity(Ethereum.SPARK_PROXY), false);
+
+        assertEq(delegator.networkLimit(subnetwork),                    0);
+        assertEq(delegator.operatorNetworkShares(subnetwork, OPERATOR), 0);
+        assertEq(delegator.hook(),                                      address(0));
+
+        assertEq(
+            delegator.hasRole(delegator.OPERATOR_NETWORK_SHARES_SET_ROLE(), RESET_HOOK),
+            false
+        );
+
+        assertEq(operatorRegistry.isEntity(Ethereum.SPARK_PROXY), false);
+
+        assertEq(
+            IOptInService(delegator.OPERATOR_NETWORK_OPT_IN_SERVICE()).isOptedIn(Ethereum.SPARK_PROXY, NETWORK),
+            false
+        );
+        assertEq(
+            IOptInService(delegator.OPERATOR_VAULT_OPT_IN_SERVICE()).isOptedIn(Ethereum.SPARK_PROXY, STAKED_SPK_VAULT),
+            false
+        );
+
+        executeAllPayloadsAndBridges();
+
+        assertEq(networkRegistry.isEntity(Ethereum.SPARK_PROXY), true);
```

Comment:
Missing tests for (or in incorrect order)
```
        delegator.setMaxNetworkLimit(0, type(uint256).max);
        slasher.setResolver(0, OWNER, "");
```

Reply:
added tests for these 2